src/main/scala/tilelink/Broadcast.scala

```diff
@@ -38,33 +132,38 @@ class TLBroadcast(params: TLBroadcastParams)(implicit p: Parameters) extends Laz
     clientFn  = { cp =>
       cp.v1copy(clients = Seq(TLMasterParameters.v1(
         name     = "TLBroadcast",
+        /* Add 2 bits encoded in [[TLBroadcastConstants]] to record transaction type for each sourceId in clients. */
         sourceId = IdRange(0, 1 << log2Ceil(cp.endSourceId*4)))))
     },
     managerFn = { mp =>
       mp.v1copy(
         endSinkId  = params.numTrackers,
         managers   = mp.managers.map { m =>
-          // We are the last level manager
+          /** This is the last level coherence manager, so it cannot be probed by mangers. */
           require (!m.supportsAcquireB)
-          // We only manage addresses which are uncached
+          /* We only manage addresses which are uncached. */
           if (m.regionType == RegionType.UNCACHED) {
-            // The device had better support line transfers
+            /* The device had better support line transfers.
+             * @todo why?
```

Comment:
Because this device is going to turn `ReleaseData`s of `lineSize` into `PutFull`s of `lineSize` and `AcquireBlocks` of `lineSize` into `Gets` of `lineSize`. So the outwards managers must visibly support such operations. This is basically checking the legality of the C->UH conversion.